modules/local/fastq_dl/main.nf

```diff
@@ -0,0 +1,43 @@
+process FASTQ_DL {
+    tag "$id"
+    label 'process_medium'
```

Comment:
Could you change to `process_single` here? There shouldn't need to be all that many resources to download files.